source/index-qan.rst

```diff
@@ -1,14 +1,6 @@
-================================================================================
-Using PMM Query Analytics
-================================================================================
+:orphan: 
```

Comment:
Some of these index files were missing so the index level was removed and all content merged into one file.